src/Spryker/Zed/ProductImage/Persistence/Propel/Schema/spy_product_image.schema.xml

```diff
@@ -41,6 +41,8 @@
         <column name="id_product_image" type="INTEGER" primaryKey="true" autoIncrement="true"/>
         <column name="external_url_small" type="VARCHAR" size="2048"/>
         <column name="external_url_large" type="VARCHAR" size="2048"/>
+        <column name="alt_text_small" type="VARCHAR" size="255" required="false"/>
+        <column name="alt_text_large" type="VARCHAR" size="255" required="false"/>
```

Comment:
ATM it's major change for the module according to our current rules (each change of DB structure must trigger major release). For minor release, the code should be written in the way the module should work correctly with the **old** DB structure **without** new fields.